rasa/core/tracker_store.py

```diff
@@ -322,7 +324,18 @@ def __init__(
         # Database might take a while to come up
         while True:
             try:
-                self.engine = create_engine(engine_url)
+                # pool_size and max_overflow can be set to control the number of
+                # connections that are kept in the connection pool. Not available
+                # for SQLite, and only  tested for postgresql. See
+                # https://docs.sqlalchemy.org/en/13/core/pooling.html#sqlalchemy.pool.QueuePool
+                if dialect == "postgresql":
+                    self.engine = create_engine(
+                        engine_url,
+                        pool_size=int(os.environ.get("SQL_POOL_SIZE", "50")),
```

Comment:
should we keep this? sounded a bit like this didn't have any effect

Reply:
yes, it does make a difference (just not quite as large as we'd hoped)